nifi-manifest/nifi-runtime-manifest/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi-manifest</artifactId>
+        <version>1.16.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>nifi-runtime-manifest</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <extension.manifest.unpack.dir>${project.build.directory}</extension.manifest.unpack.dir>
+        <build.properties.file>${project.build.directory}/classes/build.properties</build.properties.file>
+        <runtime.manifest.file>${project.build.directory}/classes/nifi-runtime-manifest.json</runtime.manifest.file>
+        <runtime.manifest.id>apache-nifi</runtime.manifest.id>
+    </properties>
+
+    <dependencies>
+        <!-- Needed to unpack the extension-manifest.xml files during the build, marked as optional
```

Comment:
I couldn't figure/track the Maven chain here, is there a chicken-and-egg-thing (with a pristine build environment) or is everything happening in the correct order?

Reply:
I have to look into why the GitHub Actions build is failing. I did a full clean build after deleting my local m2/repository/org/apache/nifi and it worked, so I thought it was good.

Reply:
Was able to straighten this out... the GA build was excluding the nifi-assembly module to save time, but my new module depends on nifi-assembly, so I created a new profile that disables the binary assembly part but still builds nifi-assembly module.